frontend/src/features/admin-form/settings/components/AuthSettingsSection/AuthSettingsSection.tsx

```diff
@@ -94,6 +97,21 @@ export const AuthSettingsSection = ({
     ][]
   }, [settings.responseMode])
 
+  const sgidTip = useMemo(
+    () =>
+      `Free Singpass authentication via Singpass app QR code login. Respondents\ 
+      must have the Singpass mobile app installed to log in and submit \
+      responses. Password login is not supported. Form admin will receive respondent's NRIC.`,
+    [],
+  )
+
+  const cpTip = useMemo(
+    () =>
+      `Corppass no longer has its own login page, and now uses Singpass to\
+      authenticate corporate users. You will still need a separate Corppass e-service ID.`,
+    [],
+  )
+
```

Comment:
The `useMemo()` calls are not needed for this. Constant strings can be assigned to `sgidTip` and `cpTip` directly, or even be moved out of the function `AuthSettingsSection()` into module constants instead.

Reply:
moved into `constants.ts`!